shared/local-debug.desktop.js

```diff
@@ -3,7 +3,7 @@ import {jsonDebugFileName} from './constants/platform.desktop'
 import {noop} from 'lodash-es'
 
 // Set this to true if you want to turn off most console logging so you can profile easier
-const PERF = false
+let PERF = false
```

Comment:
It'd be nice to add this inside `config` so the `local-debug` controls are more consistent.